src/gen/lib/Abbrev.ml

```diff
@@ -0,0 +1,169 @@
+(*
+   Abbreviations for common words, to be used in generated names.
+   These are meant to be better than truncating a word at an arbitrary
+   offset.
+*)
+
+let list = [
```

Comment:
I like it. Hopefully most grammars will use similar names.

Reply:
We can always add abbreviations for _new_ words later.